src/UCTNode.cpp

```diff
@@ -298,6 +298,7 @@ void UCTNode::accumulate_eval(float eval) {
 }
 
 UCTNode* UCTNode::uct_select_child(int color, bool is_root) {
+    (void)is_root;
```

Comment:
This line doesn't do anything does it? :)

Reply:
It suppresses the unused var warning. I wanted to keep the method signature.
We could alternatively get rid of the is_root parameter in this version.